src/main/java/com/hedvig/notificationService/customerio/ProductPricingFacadeImpl.kt

```diff
@@ -0,0 +1,26 @@
+package com.hedvig.notificationService.customerio
+
+import com.hedvig.notificationService.serviceIntegration.productPricing.client.ProductPricingClient
+import feign.FeignException
+import org.slf4j.LoggerFactory
+
+class ProductPricingFacadeImpl(private val productPricingClient: ProductPricingClient) : ProductPricingFacade {
+
+    val log = LoggerFactory.getLogger(ProductPricingFacadeImpl::class.java)
+
+    override fun getWorkspaceForMember(memberId: String): Workspace {
+
+        return try {
+            val response = productPricingClient.getContractMarketInfo(memberId)
+
+            val market = response.body!!.market
+            getWorkspaceFromMarket(market)
+        } catch (ex: FeignException) {
+            if (ex.status() != 404) {
+                log.error("Could not get contractMarketInfo for member $memberId", ex)
```

Comment:
Either like this, or don't throw anything since it's not really an error more than a member not having a contract yet. But it's up to you what you think is the intended behaviour 
```suggestion
                log.error("No contractMarketInfo available for member $memberId", ex)
```

Reply:
But we are not throwing anything, feign on the other hand throws exceptions for statuses that are note 2XX.
Maybe log it as an INFO?